includes/sanitizers/class-amp-meta-sanitizer.php

```diff
@@ -104,6 +103,17 @@ static function ( $element ) {
 		$this->re_add_meta_tags_in_optimized_order();
 	}
 
+	/**
+	 * Deduplicate the nodes of the tags that should be unique.
+	 */
+	protected function deduplicate_nodes() {
```

Comment:
I don't think this should be done here because we don't know which of the duplicated tags are the one the user intends. Otherwise, isn't it silently removed? 

Non-unique elements will be removed by the `AMP_Tag_And_Attribute_Sanitizer` here:

https://github.com/ampproject/amp-wp/blob/224c5c2f8915219447e0ea83cdd91b1db98e11a7/includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php#L638-L655

The user will be notified of the duplicates being removed.

Reply:
If we don't deduplicate here, we also can't sanitizer the attributes of these tags. Then we might add attributes to a tag only to have the `AMP_Tag_And_Attribute_Sanitizer` remove it because it was a duplicate...

Reply:
Yeah, that's tough. It's something wrestled with in #4137 as well.

What about about this: can each of the duplicates be processed here in the meta sanitizer, and then re-order them so that the one that is most correct appears first? Then the others will get removed as duplicates?

Reply:
> the one that is most correct
How would we decide on that? That might produce very complex but still brittle code.

What if you just have two wrong ones? What if you have two correct, but conflicting ones?

Reply:
I've just removed the deduplication for now, I don't think it is needed at all.